### ISSUE TITLE
[SF][A11y][MessageStrip]: repetitive screenreader announcement when link is provided

### ISSUE THREAD

**Issue by caroline-ferri** [UI5/webcomponents#13079]
### Bug Description

Our current design has a message strip that has text and a link at the end of the text. When the screenreader user focuses on the link within the message strip, the screenreader announces all the text (including the link text) to help annotate the message strip. The screenreader then repeats only the link text to give context to the actual link. The SF A11y team has reported that this repetitive announcement is an a11y violation and should be avoided. 

Please note that the SAP UI5 MessageStrip component supports a link property and does not have repeated announcement of the link text:
<img width="1552" height="717" alt="Image" src="https://github.com/user-attachments/assets/06aa3635-3e54-489e-a6f1-6494aa9b6b0d" />

SAP UI5-WC announcement:
<img width="718" height="154" alt="Image" src="https://github.com/user-attachments/assets/1ab8c454-2e29-4d74-b514-0b5168ad5f39" />

### Affected Component

MessageStrip

### Expected Behaviour

When the user navigates to a link within the message strip, the link text should only be announced once when focused. 

### Isolated Example

https://ui5.github.io/webcomponents/play/#eNq9k01vEzEQhv_K1Kc2imM4IKFlEwEFBBLQSimHSntx1pPErXe8WjsJUZT_zng_qkQUuKFEys47M-9MHnsPwpLBn5N1rJzIDoJ0hSI7Fcei9BSRIsv5hZRQO71fNX5DRi69MyDlrKD84sPN9d397UdITUlIv-A0raaFQCoEa90nX6M2HAF_8xD3DlMwysjHy8zg0hKaKziwZmxIwzIgT_iGhWPqbye0-aVvSjSy9M43UpuHTYhntZCrwf__j8srjBrKtW4CRmbw4-6TfN1i6FMJNetbi7vaN7EQ0INmcWdNXE8Nbm2Jsg3GYMlGq50MpXY4fTl50ZtFG3nmXFe1w1x1EVNWPWZ-XHizh3Y1tl7o8rE_vvafZLDVzaWUQdfvn1LXKXPVD3jm0CVSf_CpYGNfyQpD0CuUITa2BoPBroinfceVjnaLbJUbu519IaZYseQJvnUtg4Oz9Di7qZFg_u4WPvsKa87m6inXEk4mnXY28U-LdkvywZdcFCHu68Sg8mbjeCcITZlCbWnyENKOqitsASZsHUA1XOq_ohDH8WB18ioNyul7pEanLmtrEEaqIFuliwCFmKiTdNjUSW4X5Jv2e2-7wVn_W-ajdrgoPSvEMwNzC1H1xOeJ2OD3z6avzL4rFsfjL_imdUo

### Steps to Reproduce

1. open stackblitz example with JAWS
2. tab to the "Open SAP Homepage" link 
3. Note that the whole text content of the message strip is read out
4. Note that the link text is read out again
5. Navigate to the SAP UI5 Message Strip component page
6. focus on the last message strip with the link "Open SAP Homepage"
7. Note that the link text is only announced once
...

### Priority

Medium

### UI5 Web Components Version

2.18.0

### Browser

Chrome

### Operating System

macOS

### Organization

SFSF

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by s-todorova** [comment]
Hello @UI5/ui5-team-ctr-ril,

Could you please take a look at this? Issue is reproducible as described above, compare announcement on [webc sample](https://ui5.github.io/webcomponents/play/#eNq9k01vEzEQhv_K1Kc2imM4IKFlEwEFBBLQSimHSntx1pPErXe8WjsJUZT_zng_qkQUuKFEys47M-9MHnsPwpLBn5N1rJzIDoJ0hSI7Fcei9BSRIsv5hZRQO71fNX5DRi69MyDlrKD84sPN9d397UdITUlIv-A0raaFQCoEa90nX6M2HAF_8xD3DlMwysjHy8zg0hKaKziwZmxIwzIgT_iGhWPqbye0-aVvSjSy9M43UpuHTYhntZCrwf__j8srjBrKtW4CRmbw4-6TfN1i6FMJNetbi7vaN7EQ0INmcWdNXE8Nbm2Jsg3GYMlGq50MpXY4fTl50ZtFG3nmXFe1w1x1EVNWPWZ-XHizh3Y1tl7o8rE_vvafZLDVzaWUQdfvn1LXKXPVD3jm0CVSf_CpYGNfyQpD0CuUITa2BoPBroinfceVjnaLbJUbu519IaZYseQJvnUtg4Oz9Di7qZFg_u4WPvsKa87m6inXEk4mnXY28U-LdkvywZdcFCHu68Sg8mbjeCcITZlCbWnyENKOqitsASZsHUA1XOq_ohDH8WB18ioNyul7pEanLmtrEEaqIFuliwCFmKiTdNjUSW4X5Jv2e2-7wVn_W-ajdrgoPSvEMwNzC1H1xOeJ2OD3z6avzL4rFsfjL_imdUo) vs [ui5 sample](https://sapui5.hana.ondemand.com/#/entity/sap.m.MessageStrip/sample/sap.m.sample.MessageStrip).

Best,
Siyana

**Comment by ivoplashkov** [comment]
Hello @caroline-ferri,

After consulting with our accessibility experts, we have determined that this behavior does not constitute a violation. The initial announcement is triggered by the aria-label on the role="note" element, which captures the full text of the ui5-messagestrip. We cannot remove the link from this label as it often provides essential context. The subsequent announcement occurs because the link itself is the focused element. Since both the full context and the focus indicator are necessary for a compliant user experience, we consider this expected behavior.

I hope this information helps!
Best regards,
Ivaylo
